egui/src/types.rs

```diff
@@ -159,6 +159,19 @@ impl Response {
         self.ctx
             .interact_with_hovered(self.layer_id, self.id, self.rect, sense, self.hovered)
     }
+
+    /// Move the scroll to this UI.
+    /// The scroll centering is based on the `center_factor`:
+    /// * 0.0 - top  
+    /// * 0.5 - middle
+    /// * 1.0 - bottom
+    pub fn scroll_to_me(&self, center_factor: f32) {
```

Comment:
I think for now using `Align` is good enough. Keep it simple and consistent, if the necessity for more control is needed it can be changed later. :+1: